### PR TITLE
Display rockets - Lists render #13

### PR DESCRIPTION
**Pull request by Thermiee**
## Description
In this PR I displayed rockets and descriptions.

### General requirements

- [x] There are no linter errors.
- [x] Followed the **Github flow**.
- [x] The documentation is professional.

### Milestone requirements, I:

- [x] Use useSelector() Redux Hook to select the state slices and render lists of rockets in corresponding routes.

Thank you for taking the time to review this pull request. :pray:

### CODE REVIEW

File: src/components/rockets/rockets.js [hunk 1]
```diff
@@ -1,21 +1,31 @@
 import React from 'react';
-import logoImage from '../../images/planet.png';
+import PropTypes from 'prop-types';
 import './rockets.css';
 
-export default function RocketCard() {
+const RocketCard = (props) => {
```
Comment (Bavon101) [inline src/components/rockets/rockets.js:5]: - [ ] Do Style the Rocket card as per the design as per the milestone requirements, add some `margin` or `padding` between the card contents.
- [ ] The `description` also covers the entire available screen size width, please add a max-width.
_Take a look at the screenshots below_

### Required design

microverse_rocket

### Your design

emmuel

### Project requirement

design_rule

**Review by Bavon101 — CHANGES_REQUESTED** [review]
Hello @Thermiee ,
Good job so far working on the current milestone 😎,
Your project structure is professional, your commit history is also well descriptive.
You are almost there, just a few more tweaks and you're done for the millstone.

### REQUIRED CHANGES 🛑
check the comments section for the required tweaks. 🐱‍🏍 

Cheers and happy tweaking

File: src/components/rockets/rockets.js (hunk 1 above)
Comment (Thermiee) [reply src/components/rockets/rockets.js:5]: This was done in another PR

**Review by Bavon101 — APPROVED** [review]
@Thermiee 
Thanks for making the necessary changes from the [other PR](https://github.com/Bavon101/space-travelers-hub/pull/39)
 You are free to merge the current changes 🟢.

Cheers and happy coding ✔🐱‍🏍

**Pull request merged**